xml/kgraft.xml

```diff
@@ -255,6 +255,40 @@
    </procedure>
   </sect2>
  </sect1>
+ <sect1 xml:id="sec.kgraft.lifecycle">
+  <title>Patch Lifecycle</title>
+  <para>Expiration dates of life patches can be accessed with <command>zypper
+  lifecycle</command>.</para>
+  <procedure>
+   <step>
+    <para>Make sure that the package <systemitem class="resource">
```

Comment:
Use `<package>` element instead of `<systemitem>` (at least as long as you are using DocBook 5, in DocBook 4 we indeed needed to use this syntax).